src/http/routers/v1/integrations.rs

```diff
@@ -0,0 +1,188 @@
+use std::path::PathBuf;
+use std::sync::Arc;
+use axum::Extension;
+use axum::http::{Response, StatusCode};
+use tokio::sync::RwLock as ARwLock;
+use hyper::Body;
+use serde::{Deserialize, Serialize};
+use serde_json::{json, Value};
+use url::Url;
+use std::fs;
+use std::io::Read;
+#[allow(deprecated)]
+use base64::encode;
+use reqwest::Client;
+use tokio::fs as async_fs;
+use tracing::info;
+
+use crate::custom_error::ScratchError;
+use crate::global_context::GlobalContext;
+use crate::integrations::{integrations_paths, load_integration_schema_and_json, validate_integration_value, INTEGRATION_ICONS};
+use crate::yaml_configs::create_configs::{integrations_enabled_cfg, write_yaml_value};
+
+
+#[derive(Serialize, Deserialize)]
+struct IntegrationItem {
+    name: String,
+    enabled: bool,
+    schema: Option<Value>,
+    value: Option<Value>,
+}
+
+#[derive(Serialize)]
+struct IntegrationIcon {
+    name: String,
+    value: String,
+}
+
+pub async fn get_image_base64(
+    cache_dir: &PathBuf, 
+    icon_name: &str, 
+    icon_url: &str,
+) -> Result<String, String> {
```

Comment:
include base64, for example:

        let code = include_str!("alt_testsuite/py_torture1_attr.py");